RayLin.md

```diff
@@ -1,45 +1,41 @@
-# 10/03 ~ 10/14
+# 10/17 ~ 10/21
 
 - [Video Control]
   - Bug 1271765 - Visual refresh of media controls
     - review:
       - Part 1: Layout reflow fix    r?dholbert
-      - Part 2: Visul refresh        r+jaws
-      - Part 3: Tests fix            r+jaws
     - layout:
-      - discussed with :astley offline
-      - dholbert PTO til 10/17
+      - issues fixed -> revision 2
+      - dholbert PTO 10/21 - 10/25
     - a11y:
-      - study ARIA and optimize new a11y tree
-      - get help from a11y team (:marcoZ and :surkov)
-      - ask them to verify new impl if it is still accessible
-    - l10n:
-      - don't use single quote in locale string(Bug 1294275) -> workaround with JavaScript
-    - mochitest-5:
-      - system default font breaks one of case which highly rely on precise font size(width)
-      - `font-size-adjust` fix font issue on platforms individually
-    - Hope to land by the end of next week (10/21)
+      - accessible tree fixed
 
   - Bug 1302320 - add a test to confirm that video control show "noaudio" icon when video has no audio
     - patch done and pushed. Will add review flag once Bug 1271765 landed
 
-  - Next week:
-		- add the appearance tests for small size video.
-      - would be based on new layout
+  - Bug 1311700 - Add test to confirm that video control show controls in different sizes correctly
+    - have problem probing style(size, display) in mochitest, looking for similar UI test case as reference
 
+  - Bug 1312324 - Add test to confirm that fullscreen should(not) be available accordingly when video in an iframe
+    - r?jaws
+
+  - Video control test coverage enhancement tracker bug - Bug 1303958
 
 ### Video Control Planning ###
 
-- [~ late Aug, about 1 month]
-  - Visul refresh of media controls - bug 1271765
 - [after visual refresh ~]
 	- Test coverage improvement:
-		- CC button and its menu, and CC should correspond with selected one - 1303245
-		- should show "noaudio" icon if video has no audio - 1302320
-		- buttons such as play, pause, mute, CC, fullscreen should work
-		- the appearance of small size video
+		- CC button and its menu, and CC should correspond with selected one - fixed
+		- should show "noaudio" icon if video has no audio - Bug 1302320
+		- the appearance of small size video - Bug 1311700
 		- should show error state and error text should fit on most video size
+		- should show fullscreen button accordingly when video in an iframe - Bug 1312324
+		- top level synthetic media - buttons such as play, pause, mute, CC, fullscreen should work - Bug 1312334
 - [deferred]
   - Split mobile/desktop bindings - bug 1292083
-  - [De-XUL] Andriod - bug 726240
-  - Code refactor/refine
+
+
+### Autofill Planning ###
+
+  - [November ~]
+    - Study autocomplete && passwordmanager related code
```

Comment:
I would recommend tackle problems by actually taking on issues instead of "study". It's really hard to measure (for oneself) the expected result of the "study".